demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -148,6 +150,40 @@ def is_insecure_configured_correctly(self):
             return self.is_valid_param(insecure_field_name, 'Trust any certificate (not secure)')
         return True
 
+    def is_checkbox_param_configured_correctly(self):
+        # type: () -> bool
+        """Check that if an integration has a checkbox parameter it is configured properly.
+        Returns:
+            bool. True if the checkbox parameter is configured correctly, False otherwise.
+        """
+        err_msgs = []
+        configuration = self.current_file.get('configuration', [])
+        for configuration_param in configuration:
+            param_name = configuration_param['name']
+            if configuration_param['type'] == 8 and param_name not in ('insecure', 'unsecure', 'proxy', 'isFetch'):
+                err_msg = self.is_valid_checkbox_param(configuration_param, param_name)
+                if err_msg:
+                    err_msgs.append(err_msg)
```

Comment:
let's not aggregate, let's just print them out - can be done in the other funciton

Reply:
done